test/stress/thousand_deployments_test.go

```diff
@@ -114,7 +114,7 @@ func thousandDeploymentsTest(ctx context.Context, c client.Client, invConfig inv
 		Expect(e.ErrorEvent.Err).To(BeNil())
 	}
 	for _, e := range destroyerEvents {
-		Expect(e.PruneEvent.Error).To(BeNil(), "PruneEvent: %v", e.PruneEvent)
+		Expect(e.DeleteEvent.Error).To(BeNil(), "DeleteEvent: %v", e.DeleteEvent)
```

Comment:
Why change `PruneEvent` to `DeleteEvent`?

Is there a reason that we run the same for loop 3 times instead of combining them?

Reply:
> Why change PruneEvent to DeleteEvent?

For the Destroyer there are no PruneEvents. It replaces them with DeleteEvents. The PruneEvents are only used by the Applier. This change fixes the test to catch the right event errors. 

Personally, I think they should all be DeleteEvents, since there's no real difference between them, but that's tech debt from years ago that we haven't gotten around to cleaning up.

> Is there a reason that we run the same for loop 3 times instead of combining them?

No particular reason. That was just the existing behavior. I could clean it up like I did in the other test. That would make it easier to debug by printing the whole event list.

Reply:
added an function to clean it up

Reply:
Yeah, distinguishing between PruneEvent and DeleteEvent seemed like a good idea at the time. But agree with @karlkfi that they should all just be DeleteEvents.